docs/ADS1115--ADC0_ADC1_ADC2-infos.md

```diff
@@ -23,6 +23,12 @@ Usage in car hull:
 
 ![img](.ADS1115--ADC0_ADC1_ADC2-infos/FSR-CurcuitDiagram.drawio.png)
 
+### Board Assembly
+
+![image-20210823130410331](.ADS1115--ADC0_ADC1_ADC2-infos/image-20210823130410331.png)
+
+![image-20210823130639854](.ADS1115--ADC0_ADC1_ADC2-infos/image-20210823130639854.png)
+
```

Comment:
Please rename and add them to a non-hidden folder.

Reply:
done